packages/main/src/Input.ts

```diff
@@ -1804,6 +1804,7 @@ class Input extends UI5Element implements SuggestionComponent, IFormInputElement
 			suggestionPopoverHeader: {
 				"display": this._listWidth === 0 ? "none" : "inline-block",
 				"width": this._listWidth ? `${this._listWidth}px` : "",
+				"max-width": this._inputWidth && (this._inputWidth / remSizeIxPx) > 40 ? `${this._inputWidth}px` : "40rem",
```

Comment:
Instead of calculating the max-width of the header this way, you can set max-width: "100%" and the header will expand to its container width

Please, also fix the typo "remSizeIxPx" to be "remSizeInPx" because it is used in the code below